modules/logging-loki-retention.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+:_content-type: REFERENCE
+[id="logging-loki-retention_{context}"]
+
+.Example LokiStack CR 
+
+[source,yaml]
+----
+apiVersion: loki.grafana.com/v1
```

Comment:
This look like the correct LokiStack for 'per-tenant stream based retention'. We can also define global stream based retention' for LokiStack.

For e.g.
 ```
apiVersion: loki.grafana.com/v1
 kind: LokiStack
 metadata:
   name: lokistack-sample
   namespace: openshift-logging
 spec:
   limits:
     global:
       retention:
         days: 2
         streams:
         - days: 4
           priority: 1
           selector: '{kubernetes_namespace_name=~"test.+"}'
         - days: 1
           priority: 1
           selector: '{log_type="application"}'
         - days: 15
           priority: 1
           selector: '{log_type="audit"}'
   managementState: Managed
   replicationFactor: 1
   size: 1x.small
   storage:
    schemas:
    - effectiveDate: "2020-10-11"
       version: v11
    secret:
      name: gcs-secret
      type: gcs
  storageClassName: standard
  tenants:
    mode: openshift-logging
```